### DISCUSSION TITLE
How to manage shared assets (images/fonts/etc)?

### DISCUSSION THREAD

**Discussion by mschilling** [framna-nl-archive/krew-workspace#63] [Q&A]
Alle 4 frontend apps hebben in de basis dezelfde assets nodig, zoals images en fonts. 

Daarom een paar vragen:
1. Hoe willen we de assets folder inrichten/organiseren?
2. Welke bestand types willen we hier onder brengen

En stel dat we bijvoorbeeld het Krew logo, of splash screen image, in verschillende formaten willen opslaan .. hoe ziet de structuur er dan uit?

Linked issues:
* #62
* #48

**Comment by aaltw** [comment]
Voorstel: gebruik https://heroicons.com/

Reden: De huidige iconen set die gebruikt wordt lijkt hier heel erg op, en zoals ze nu ge-exporteert worden uit Sketch worden we ook verdrietig van.
